aim/engine/utils.py

```diff
@@ -121,7 +121,12 @@ def is_pytorch_module(inst):
     """
     return inst_has_typename(inst, ['torch', 'Module'])
 
-
+def is_tensorflow_session(inst):
+    """
+    Check whether `inst` is isntance of tensorflow session
+    """
+    return inst_has_typename(inst, ['session'])
```

Comment:
Is only checking session enough? Maybe we should also check if an instance has `tf`, `tensorflow` in its class type

Reply:
Added 'tensorflow' to types.